### ISSUE TITLE
Angular 6 issue

### ISSUE THREAD

**Issue by FrciSmrci** [SortableJS/angular-sortablejs#112]
Hi @smnbbrv 

I encountered an problem with the latest version of the library and Angular 6, it works fine in versions <6. When I drag and drop the last item in the array an new undefined item is created and pushed into the array. Here is a [video](https://drive.google.com/open?id=1os0gbaspfUP0a75bc7bAdOaWtTokwmqD) demonstrating the issue. Debugging suggests that the problem occurs inside the sortable directive in the function proxyEvent at this line `(_a = this.optionsWithoutEvents)[eventName].apply(_a, params);` 

I didn't go further into, do you have any idea what could be the problem? 

Thank you in advance.

Have a great day!

**Comment by smnbbrv** [comment]
Hi there

Could you please reproduce the problem on stackblitz? I use it on angular 6 project and everything seems to be fine so far

**Comment by FrciSmrci** [comment]
Hi @smnbbrv , 

so in the first run I changed my sortable input items to array of strings and removed all of the custom options to try it out on an simplified example. Because that didn't work I thought that the problem must lay in the library somehow. 

Turns out that the problem was an separated tiny icon wrapper inside the [sortablejs] declared div which was set to position absolute and carried an icon with position relative.

Thanks for the help. 😅 

Have a great day!
